frontend/vue/components/OneXP/OneXPBanner.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="one-xp-banner">
+    <div class="one-xp-banner__container">
+      <span
+        class="one-xp-banner__copy"
+        v-text="
+          `🎉  Qiskit is getting a new learning experience on IBM Quantum!`
+        "
+      />
+      <a
+        class="one-xp-banner__cta"
+        href="https://example.com"
+        rel="noopener noreferrer"
+        target="_blank"
+        @click="onClick"
+      >
+        <span class="one-xp-banner__cta__label" v-text="`Check it out!`" />
+        <LaunchIcon class="one-xp-banner__cta__icon" />
+      </a>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
+import LaunchIcon from '@carbon/icons-vue/lib/launch/16'
+
+@Options({
+  components: { LaunchIcon }
+})
+export default class OneXPBanner extends Vue {
+  onClick () {
+    this.$trackClickEvent('go-to-1xp-learning', '1xp-banner')
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "carbon-components/scss/globals/scss/typography";
+@import "carbon-components/scss/globals/scss/layout";
+@import "../../../scss/variables/colors.scss";
+@import "~/../scss/variables/mq.scss";
+
+.one-xp-banner {
+  @include type-style("body-long-01");
+
+  background-image: linear-gradient(to right, $purple-10, #fff);
+  border-bottom: 1px solid $border-color;
+  padding: $spacing-05 $spacing-06;
+
+  &__container {
+    display: inline-flex;
+    flex-direction: column;
+    gap: $spacing-03;
+
+    @include mq($from: large) {
+      flex-direction: row;
+    }
+  }
+
+  &__copy {
+    font-weight: bold;
```

Comment:
same comment as in another PR, but this seems to be spec'd out to be 600...

Reply:
Similarly,

> As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.